lib_standard_app/Makefile.standard_app

```diff
@@ -0,0 +1,93 @@
+#*******************************************************************************
+#   Ledger SDK
+#   (c) 2022 Ledger
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#*******************************************************************************
+
+STANDARD_APP_FLAGS = 0x000
```

Comment:
could we do something relating to `APPLICATION_FLAG_LIBRARY` so that it can be enabled more easily?

Reply:
Nope :D